packages/testing/src/fixture.ts

```diff
@@ -199,9 +199,21 @@ export class Fixture {
   }
 
   async runWithBuild(): Promise<string> {
+    const { output } = await this.runWithBuildAndIntrospect()
+    return output
+  }
+
+  async runWithBuildAndIntrospect() {
```

Comment:
This lets us get rid of snapshot tests in an incremental way. Rather than using `runWithBuild()`, which returns only the stdout + stderr output, we can use `runWithBuildAndIntrospect()` and assert things like the mutated config, success state, etc.

As part of this, I have completely removed snapshots from the deploy configuration API tests. ⚰️

Reply:
🔥